GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.ts

```diff
@@ -189,6 +189,120 @@ namespace gdjs {
       gameCanvas.focus();
     }
 
+    useCanvas(gameCanvas: HTMLCanvasElement): void {
```

Comment:
There is a lot of copy from the `createStandardCanvas` function. Could you rework `createStandardCanvas` so that it's using `useCanvas` under the hood?

Also, the name is probably not showing this is an important operation enough, that just can't be redone a second time. So I think it should be called `initializeForCanvas`

Reply:
Thank you for review! I've refactored  `createStandardCanvas`, renamed `useCanvas`, updated `runtimegame.dispose()` and added tests.